Explorer/Assets/DCL/AvatarRendering/Emotes/Helpers/MemoryEmotesCache.cs

```diff
@@ -37,40 +49,59 @@ public bool TryGetEmote(URN urn, out IEmote emote)
         public void Unload(IPerformanceBudget frameTimeBudget)
         {
             for (LinkedListNode<(URN key, long lastUsedFrame)> node = listedCacheKeys.First; frameTimeBudget.TrySpendBudget() && node != null; node = node.Next)
-                if (emotes.TryGetValue(node.Value.key, out IEmote emote))
-                    if (TryUnloadAllWearableAssets(emote))
-                    {
-                        emotes.Remove(node.Value.key);
-                        cacheKeysDictionary.Remove(node.Value.key);
-                        listedCacheKeys.Remove(node);
-                    }
+            {
+                URN urn = node.Value.key;
+
+                if (!emotes.TryGetValue(urn, out IEmote emote))
+                    if (emotes.TryGetValue(urn.ToLower(), out emote))
+                        urn = urn.ToLower();
+
+                if (emote == null) continue;
```

Comment:
Is it possible to save an null emote? Should we stop that in the Set()?

Reply:
It is a safe check in case the emote was previously removed or not existent by any reason